.github/workflows/codeql.yml

```
@@ -19,6 +19,11 @@ env:
 jobs:
   # Identify the CodeQL tool versions to use in the analysis job.
   check-codeql-versions:
+    strategy:
+      fail-fast: false
+      matrix:
+        language: javascript, actions
```

```suggestion
        language: ['javascript', 'actions']
```

I think it's not liking the syntax: https://github.com/github/codeql-action/actions/runs/12957765224/workflow

---

Yep!